missing_data/missing_data.Rmd

```diff
@@ -0,0 +1,276 @@
+
+When researchers encounter missingness in an experiment, they often examine the relationship between missingness of outcomes and treatment assignment. We have denoted reporting (or non-missingness) as a potential outcome of treament assignment, $R_i(Z)$. For a binary treatment, we can denote four "types" of subjects in the experiment, as denoted in the following table. 
+
+| Type                  | Proportion | $R_i(1)$ | $R_i(0)$ |
+|-----------------------|:-------:|:--------:|:--------:|
+| Always Reporter       | $\pi_A$ |     1    |     1    |
+| If Treated Reporter   | $\pi_T$ |     1    |     0    |
+| If Untreated Reporter | $\pi_U$ |     0    |     1    |
+| Never Reporter        | $\pi_N$ |     0    |     0    |
+
+In the case in which missingness is related to potential outcomes but not to treatment assignment, we are generally not able to identify an unbiased estimate of the ATE. However we may be interested in identifying the LATE among always reporters -- those subjects for which we observe the outcome regardless of treatment assignment. This effect can be a main estimand of interest and even the most policy-relevant estimand in certain settings. However, to estimate a LATE, we want to be sure that the outcomes that we observe are those of always reporters, not if-treated or if-untreated reporters. 
+
+To assess the plausibility of ths conjecture, we often examine the relationship between treatment assignment and reporting (non-missingness). Using the notation from the table, we can estimate:
+
+\begin{align}
+E[R_i(1)-R_i(0)] &= \pi_A + \pi_T - (\pi_A + \pi_U)\\
+&=\pi_T - \pi_U
+\end{align}
+
+If we find no difference in reporting across treatment groups, this test provides no evidence that $\pi_T \neq \pi_U$. However, to further justify that the complete observations in the data are those of "always reporters," we further must know that $\pi_T = \pi_U = 0$. We cannot test this by examining the relationship between reporting and treatment assignment. As such, we generally complement this test with a justification of why reporting should not be endogenous to treatment assignment. If reporting is not endogenous, then in principle the assumption that $\pi_T = \pi_U = 0$ holds. As such, if we find no evidence of differential levels of reporting and see no way that reporting should be endogenous to treatment, we can justify that the $E[Y_i(1)|R_i(1) = 1] - E[Y_i(0)|R_i(0) = 1]$ estimates the LATE among always reporters.
+
+[Jake, I think you wanted this as a justification of MIPO too (when adding covariates) and doing RI on f-statistics or the like. Do you still want this?]
+
+
+##7. What is imputation?
+
+The most common approaches used to deal with missing data involve the imputation or "filling in" of missing values. In the following dataset with missingness, imputation procedures "fill in" the missing data -- the `NA`s. 
+
+```{r, warning = F, message = F, echo = F}
+set.seed(1234)
+X <- rnorm(12)
```

Comment:
```suggestion
X <- signif(rnorm(12),4)
```

Reply:
Just trying to make the table a little easier to read --- no need for so many digits. Maybe better to do this in `kable()`.